src/neptune_tensorboard/integration/tensorflow_integration.py

```diff
@@ -231,37 +110,5 @@ def text(name, data, step=None, description=None):
     tf.summary.text = text
     tf.summary._original_no_neptune_text = _text
 
-    # Tensorflow 2.3 renames the internal method from `_log_metrics` to `_log_epoch_metrics`
-    # and changes its parameters. The conditional below handles both versions.
-    if hasattr(tf.keras.callbacks.TensorBoard, "_log_metrics"):
-
-        _tb_log_metrics = tf.keras.callbacks.TensorBoard._log_metrics
-
-        def _log_metrics(instance, logs, prefix, step):
-            exp = experiment_getter()
-            for (name, value) in logs.items():
-                if name in ("batch", "size", "num_steps"):
-                    continue
-                exp.log_metric(get_channel_name(name), x=step, y=value)
-
-            _tb_log_metrics(instance, logs, prefix, step)
-
-        tf.keras.callbacks.TensorBoard._log_metrics = _log_metrics
-
-    elif hasattr(tf.keras.callbacks.TensorBoard, "_log_epoch_metrics"):
-
-        _tb_log_epoch_metrics = tf.keras.callbacks.TensorBoard._log_epoch_metrics
-
-        def _log_epoch_metrics(instance, epoch, logs):
-            exp = experiment_getter()
-            for (name, value) in logs.items():
-                if name in ("batch", "size", "num_steps"):
-                    continue
-                exp.log_metric(get_channel_name(name), x=epoch, y=value)
-
-            _tb_log_epoch_metrics(instance, epoch, logs)
-
-        tf.keras.callbacks.TensorBoard._log_epoch_metrics = _log_epoch_metrics
-
-    else:
-        raise AttributeError("Unsupported Tensorflow version")
+    tf.summary.graph = graph
+    tf.summary._original_no_neptune_graph = _graph
```

Comment:
Is there any good reason for storing it?

Reply:
Currently, no good reason.